src/test/java/org/openlmis/buq/validate/BottomUpQuantificationValidatorTest.java

```diff
@@ -58,32 +57,6 @@ public void shouldThrowExceptionIfStatusIsNotDraftForSubmission() {
     validator.validateCanBeSubmitted(BottomUpQuantificationDto.newInstance(target), targetId);
   }
 
-  @Test(expected = ValidationMessageException.class)
-  public void shouldThrowExceptionIfStatusIsNotSubmittedForAuthorization() {
-    UUID targetId = UUID.randomUUID();
-    BottomUpQuantification target = new BottomUpQuantificationDataBuilder()
-        .withStatus(BottomUpQuantificationStatus.DRAFT)
-        .build();
-
-    when(quantificationService.findBottomUpQuantification(targetId))
-        .thenReturn(target);
-    when(quantificationService.canSkipAuthorization(target))
-        .thenReturn(false);
-
-    validator.validateCanBeAuthorized(BottomUpQuantificationDto.newInstance(target), targetId);
-
-    verify(quantificationService).canSkipAuthorization(target);
-  }
-
-  @Test(expected = ValidationMessageException.class)
-  public void shouldThrowExceptionIfStatusIsNotAuthorizedForRejection() {
-    BottomUpQuantification target = new BottomUpQuantificationDataBuilder()
-            .withStatus(BottomUpQuantificationStatus.IN_APPROVAL)
-            .build();
-
-    validator.validateCanBeRejected(target);
-  }
-
```

Comment:
those are not true anymore - the validation exception should not be thrown in those cases.